airbyte-cdk/python/airbyte_cdk/sources/streams/http/vcr_helper.py

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+import json
+
+import yaml
+from airbyte_cdk.utils.airbyte_secrets_utils import filter_secrets
+
+# Use the libYAML versions if possible
+try:
+    from yaml import CDumper as Dumper
+    from yaml import CLoader as Loader
+except ImportError:
+    from yaml import Dumper, Loader
+
+import vcr
+
+
+class VcrHelper:
+    def __init__(self, path_to_test_fixture=None):
+        if path_to_test_fixture is not None:
+            with open(path_to_test_fixture, "r") as f:
+                self.test_fixture = json.loads(f.read())
+        else:
+            self.test_fixture = {"filter_headers": {}}
+
+    def get_vcr(self):
+        my_vcr = vcr.VCR()
+        my_vcr.register_serializer("no_secrets", self)
+        return my_vcr
+
+    def get_filter_headers(self):
+        return self._get_filters("filter_headers")
+
+    def get_filters(self):
```

Comment:
this enables us to filter specific headers or request params even if it's not indicated as being an airbyte secret